File: androidunsplash/src/main/java/com/kc/unsplash/models/Portfolio.java
package com.kc.unsplash.models;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class Portfolio {

    @SerializedName("url")
    @Expose
    private String url;

}

File: androidunsplash/src/main/java/com/keenencharles/unsplash/models/Portfolio.kt
package com.keenencharles.unsplash.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Portfolio(
        @SerializedName("url") val url: String? = null
): Parcelable